composer.json

```diff
@@ -21,7 +21,7 @@
         "doctrine/common": "~2.0"
     },
     "require-dev": {
-        "symfony/symfony": "^2.7|^3.3|^4.0",
+        "symfony/symfony": "^2.7.1|^3.3|^4.0",
```

Comment:
The `^2.7.1` part is valid for Symfony 2.7 and 2.8: https://semver.mwl.be/#?package=symfony%2Fsymfony&version=%5E2.7.1%7C%5E3.3%7C%5E4.0&minimum-stability=stable

Reply:
2.7.0 doesn’t work?

Reply:
Unfortunately, Symfony 2.7.0 was causing errors in the `--prefer-lowest` job because of deprecations: https://travis-ci.org/liip/LiipFunctionalTestBundle/builds/313161686